docs/aerodromes/classd/Moorabbin.md

```diff
@@ -8,9 +8,23 @@
 | Name               | Callsign       | Frequency        | Login ID                         |
 | ------------------ | -------------- | ---------------- | ---------------------------------------- |
 | **Moorabbin ADC**  | **Moorabbin Tower**  | **118.100**         | **MB_TWR**                        |
+| Moorabbin ADC 2†  | Moorabbin Tower  | 123.000        | MB-2_TWR                        |
 | **Moorabbin SMC**   | **Moorabbin Ground**   | **134.250**          | **MB_GND**                       |
 | Moorabbin ATIS        |                | 120.900          | YMMB_ATIS                               |
 
+† *Non-standard positions* may only be used in accordance with [VATPAC Air Traffic Services Policy](https://vatpac.org/publications/policies){target=new}
+
+### Dual ADC Operations
+When Moorabbin ADC 2 is online, responsibility for the **Runway**, **Circuit**, and **Airspace** is divided between the two ADC controllers.
+
+ADC 1 takes responsibility for the East Runway, Circuit and Airspace (Runway **17L/35R** and **13L/31R**).
+
+ADC 2 takes responsibility for the West Runway, Circuit and Airspace (Runway **17R/35L** and **13R/31L**).
+
+ADC 2 is **not permitted** online when *Single Runway* Operations are in use.
```

Comment:
Would there ever be single runway use at MB? 

runway 22/04 is never used, only for requests and operational need (emergencies) and is not allowed to be a default. 

I would say better wording would be if ADC 1 is online or something?

Reply:
> Would there ever be single runway use at MB? 

Yep, at night.